x/merkledb/db.go

```diff
@@ -679,16 +690,16 @@ func (db *merkleDB) NewView(_ context.Context, batchOps []database.BatchOp) (Tri
 // Returns a new view that isn't tracked in [db.childViews].
 // For internal use only, namely in methods that create short-lived views.
 // Assumes [db.lock] isn't held and [db.commitLock] is read locked.
-func (db *merkleDB) newUntrackedView(batchOps []database.BatchOp) (*trieView, error) {
-	if db.closed {
-		return nil, database.ErrClosed
-	}
```

Comment:
Kind of weird that this was removed - but because this is an internal function it makes sense that this should never really happen anyways

Reply:
we always check it before we call this function, so it couldn't really happen as you said.